moveit_planners/pilz_industrial_motion_planner/src/cartesian_limits_parameters.yaml

```diff
@@ -0,0 +1,25 @@
+cartesian_limits:
+  max_trans_vel: {
+    type: double,
+    # default_value: ,
+    description: "Max translation velocity",
+    # validation:
+  }
+  max_trans_acc: {
+    type: double,
+    # default_value: ,
+    description: "Max translation acceleraion",
+    # validation:
+  }
+  max_trans_dec: {
+    type: double,
+    # default_value: ,
+    description: "Max translation decceleration",
+    # validation:
+  }
+  max_rot_vel: {
+    type: double,
+    # default_value: ,
+    description: "Max rotational velocity",
+    # validation:
+  }
```

Comment:
Do these have any defaults or validations that would make sense for them?

Reply:
I was thinking of leaving the defaults empty so that the user would be forced to set values. Do you think that's a good approach?
Otherwise I can default the defaults to 0 and figure out another way to ensure the use overrides them

Reply:
My comment was more that we should not leave commented-out lines here. I was asking if you planned on adding defaults. If we need the user to specify the values, not having a default is the right choice.